src/staticarrays_support.jl

```diff
@@ -0,0 +1,10 @@
+import StaticArrays: StaticArray, tuple_prod
+
+@generated function StructArrays.staticschema(::Type{s}) where {s <: StaticArray{S,T,N}} where {S,T,N}
```

Comment:
```suggestion
@generated function StructArrays.staticschema(::Type{<:StaticArray{S, T}}) where {S, T}
```

I think this should work and be a bit more readable (mini note, sorry to be a pain, this package uses spaces after commas between type parameters).

Reply:
Not at all, this is helpful to see how other devs do things. Thanks for the recommendations!